src/Nethermind/Nethermind.Merge.Plugin/Data/ClientVersionV1.cs

```diff
@@ -0,0 +1,31 @@
+// SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using Nethermind.Core;
+
+namespace Nethermind.Merge.Plugin.Data;
+
+/// <summary>
+///   The client version specification.
+///   <seealso cref="https://github.com/ethereum/execution-apis/pull/517/files?short_path=f1e647c#diff-f1e647ce063c92e6fd6cd448746b1d1effcfd2fa2e1b031a71f8ce2f74ba0952"/>
+/// </summary>
+public readonly struct ClientVersionV1
+{
+    public ClientVersionV1()
+    {
+        this.code = ProductInfo.ClientCode;
+        this.name = ProductInfo.Name;
+        this.version = ProductInfo.Version;
+        this.commit = ProductInfo.Commit;
```

Comment:
```suggestion
        Code = ProductInfo.ClientCode;
        Name = ProductInfo.Name;
        Version = ProductInfo.Version;
        Commit = ProductInfo.Commit;
```
No need for this before the assignment